zorg/buildbot/builders/ClangBuilder.py

```diff
@@ -494,103 +500,153 @@ def _getClangCMakeBuildFactory(
                                    workdir=stage2_build,
                                    env=env))
 
-        # Get generated python, lnt
-        python = util.Interpolate('%(prop:builddir)s/test/sandbox/bin/python')
-        lnt = util.Interpolate('%(prop:builddir)s/test/sandbox/bin/lnt')
-        lnt_setup = util.Interpolate('%(prop:builddir)s/test/lnt/setup.py')
+        if testWithLNT:
+            # Get generated python, lnt
+            python = util.Interpolate('%(prop:builddir)s/test/sandbox/bin/python')
+            lnt = util.Interpolate('%(prop:builddir)s/test/sandbox/bin/lnt')
+            lnt_setup = util.Interpolate('%(prop:builddir)s/test/lnt/setup.py')
+
+            # Get latest built Clang (stage1 or stage2)
+            cc = util.Interpolate(f'%(prop:builddir)s/{compiler_path}/bin/{cc}')
+            cxx = util.Interpolate(f'%(prop:builddir)s/{compiler_path}/bin/{cxx}')
+
+            # LNT Command line (don't pass -jN. Users need to pass both --threads
+            # and --build-threads in nt_flags/test_suite_flags to get the same effect)
+            use_runtest_testsuite = len(nt_flags) == 0
+            if not use_runtest_testsuite:
+                test_suite_cmd = [python, lnt, 'runtest', 'nt',
```

Comment:
I was looking at this earlier, wondering what nt even is. Nightly test?

The logic is a bit convoluted here but I see you're porting the existing style so it's ok for now.

Reply:
No one in zorg ever passes nt_flags, but this PR wouldn't be the time to remove it anyway.

Reply:
LNT has a nt (nightly test) configuration. More on it can be found here https://llvm.org/docs/lnt/tests.html. I have not tested this LNT configuration myself.